File: batchbhavna/FirstApplication/app/src/main/java/com/example/firstapplication/fragments/SecondFragment.java
package com.example.firstapplication.fragments;

import android.content.Context;
import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import android.util.Log;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.example.firstapplication.R;


public class SecondFragment extends Fragment {
    private static final String TAG = "SecondFragment";

    TextView textView;
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        Log.e(TAG, "onCreateView: " );
        return inflater.inflate(R.layout.fragment_second, container, false);


    }
    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        String value = getArguments().getString("first");
        Log.e(TAG, "onViewCreated: " );
        textView = view.findViewById(R.id.textsecond);

        textView.setText(value);

    }

    @Override
    public void onDestroy() {
        super.onDestroy();
        Log.e(TAG, "onDestroy: " );
    }

    @Override
    public void onAttach(@NonNull Context context) {
        super.onAttach(context);
        Log.e(TAG, "onAttach: " );
    }

    @Override
    public void onStart() {
        super.onStart();
        Log.e(TAG, "onStart: " );
    }

    @Override
    public void onResume() {
        super.onResume();
        Log.e(TAG, "onResume: " );
    }

    @Override
    public void onPause() {
        super.onPause();
        Log.e(TAG, "onPause: " );
    }

    @Override
    public void onStop() {
        super.onStop();
        Log.e(TAG, "onStop: " );
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
        Log.e(TAG, "onDestroyView: " );
    }

    @Override
    public void onDetach() {
        super.onDetach();
        Log.e(TAG, "onDetach: " );
    }
}

File: 4pmclass/MyFirstApplication/app/src/main/java/com/example/myfirstapplication/SecondFragment.kt
package com.example.myfirstapplication

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.util.Log
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import android.widget.TextView


class SecondFragment : Fragment() {
lateinit var button:Button
    lateinit var movetorelative:Button
    lateinit var textView: TextView
var strinvalue :String? = null
    var TAG :String = "Second Fragment"
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        Log.e(TAG,"OnCreateView Called")

        return inflater.inflate(R.layout.fragment_second, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        strinvalue = arguments!!.getString("first")
        textView= view.findViewById(R.id.dataget)
        textView.text = strinvalue
        button = view.findViewById(R.id.move_to_FirstFragment)
        button.setOnClickListener {
            val fragmentManager = fragmentManager
            val fragmentTraction = fragmentManager!!.beginTransaction();
            fragmentTraction.replace(R.id.fragmentcontainer,FirstFragment());
            fragmentTraction.commit()
        }
        movetorelative = view.findViewById(R.id.move_to_relativelayout)
        movetorelative.setOnClickListener {
            val intent = Intent(view.context,RelativeExamples::class.java)
            startActivity(intent)
        }
        Log.e(TAG,"OnView Created Called")

    }
    override fun onAttach(context: Context) {
        super.onAttach(context)
        Log.e(TAG,"OnAttach Called")
    }

    override fun onDestroy() {
        super.onDestroy()
        Log.e(TAG,"OnDestroy");
    }

    override fun onStart() {
        super.onStart()
        Log.e(TAG,"OnStart");
    }

    override fun onResume() {
        super.onResume()
        Log.e(TAG,"OnDResumen");
    }

    override fun onPause() {
        super.onPause()
        Log.e(TAG,"OnPause");
    }

    override fun onStop() {
        super.onStop()
        Log.e(TAG,"OnStopd");
    }

    override fun onDestroyView() {
        super.onDestroyView()
        Log.e(TAG,"OnDestroyView");
    }

    override fun onDetach() {
        super.onDetach()
        Log.e(TAG,"OnDettach");
    }
}